platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -363,6 +363,8 @@ function _getViewportComponent(displaySets, viewportComponents) {
       return component;
     }
   }
+  console.warn("Didn't find a viewport component handling", SOPClassHandlerId);
```

Comment:
This prevents crashing when viewing a study with DICOM SR in it.

Reply:
Same for this comment